timeseries/src/autogluon/timeseries/evaluator.py

```diff
@@ -75,6 +89,7 @@ class TimeSeriesEvaluator:
         * ``MSE``: mean squared error
         * ``RMSE``: root mean squared error
         * ``WAPE``: weighted absolute percentage error. See https://docs.aws.amazon.com/forecast/latest/dg/metrics.html#metrics-WAPE
+        * ``RMSSE``: Root Mean Squared Scaled Error . See https://mofc.unic.ac.cy/m5-competition/
```

Comment:
- Maybe we can point to https://otexts.com/fpp3/accuracy.html#scaled-errors for both RMSSE and MASE?
- Can you please add the documentation also to `TimeSeriesPredictor`?

Reply:
Agreed and updated.